### PR TITLE
KOGITO-2870: Move graphQL queries from Task Console to common module

### PR DESCRIPTION
**Pull request by pefernan**
I wanted to make this move as a part of another PR but I thought it would make the PR harder to review. 

@cristianonicolai @Sara4994 @jstastny-cz could you please take a look?

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: ui-packages/packages/common/src/graphql/queries.tsx [hunk 1]
```diff
@@ -243,3 +243,61 @@ const GET_INPUT_FIELDS_FROM_TYPES = gql`
     }
   }
 `;
+
+// @ts-ignore
```
Comment (cristianonicolai) [inline ui-packages/packages/common/src/graphql/queries.tsx]: no need to add `// @ts-ignore` as file is already ignored.

**Review by jstastny-cz — APPROVED** [review]

**Comment by pefernan** [comment]
jenkins retest this please

**Review by Sara4994 — APPROVED** [review]

**Comment by Sara4994** [comment]
looks good :+1:

**Comment by pefernan** [comment]
jenkins retest this please

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**